resque-durable.gemspec

```diff
@@ -5,11 +5,6 @@ Gem::Specification.new do |s|
   s.summary  = 'Resque queue backed by database audits, with automatic retry'
   s.homepage = 'https://github.com/zendesk/resque-durable'
   s.license  = 'MIT'
-  s.files    = [
-    'lib/resque/durable.rb',
-    'lib/resque/durable/guid.rb',
-    'lib/resque/durable/monitor.rb',
-    'lib/resque/durable/queue_audit.rb'
-  ]
+  s.files    = `git ls-files lib`.split($/)
```

Comment:
I actually don't like using git for this, but I think that's personal preference.